### PR TITLE
feat: script to print balances of important keys

### PR DESCRIPTION
**Pull request by paulbalaji**
resolves https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4349

- feat: script to print balances of important keys
- super useful as a pre-deploy spot check

e.g. on testnet4
image

### CODE REVIEW

File: typescript/infra/scripts/print-balances.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    process.exit(1);
+  });
+
+function convertToNativeTokenValue(value: BigNumber, decimals: number): string {
+  const divisor = BigNumber.from(10).pow(decimals);
+  const integerPart = value.div(divisor);
+  const fractionalPart = value.mod(divisor).toString().padStart(decimals, '0');
+  const truncatedFractionalPart = fractionalPart.slice(0, 3);
+  return `${integerPart.toString()}.${truncatedFractionalPart}`;
+}
```
Comment (paulbalaji) [inline typescript/infra/scripts/print-balances.ts:82]: there's surely a better way ?

Comment (Mo-Hussain) [reply typescript/infra/scripts/print-balances.ts:82]: Does ethers have some nice helpers?

Comment (Mo-Hussain) [reply typescript/infra/scripts/print-balances.ts:82]: https://docs.ethers.org/v5/api/utils/display-logic/#utils-formatUnits

**Review by Mo-Hussain — APPROVED** [review]

File: typescript/infra/scripts/print-balances.ts (hunk 1 above)
Comment (paulbalaji) [reply typescript/infra/scripts/print-balances.ts:82]: hmmm it tidies up the code but need to figure out how to truncate 
image

Comment (paulbalaji) [reply typescript/infra/scripts/print-balances.ts:82]: nvm got it

**Pull request merged**